libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -84,6 +86,19 @@ const AP_Param::GroupInfo AP_UAVCAN::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SRV_RT", 4, AP_UAVCAN, _servo_rate_hz, 50),
 
+    // @Param: IMUD_M
```

Comment:
I'd prefer _BM suffix, to match the other two bitmask vars in the same object

Reply:
I don't believe it will fit.